packages/shorebird_cli/lib/src/extensions/organization.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:shorebird_code_push_protocol/shorebird_code_push_protocol.dart';
+
+/// {@template organization_display}
+/// Returns the user-facing display name of an [Organization].
+/// {@endtemplate}
+extension OrganizationDisplay on Organization {
+  /// {@macro organization_display}
+  String get displayName => switch (organizationType) {
+        OrganizationType.team => name,
+        OrganizationType.personal => 'Personal',
```

Comment:
I’d use the user display name and fallback to email instead of “Personal” here to be consistent with the console.

Reply:
![Screenshot 2024-10-08 at 2 54 48 PM](https://github.com/user-attachments/assets/9d11509a-98d4-4c34-9a52-cdf33bdd929a)

So something like this, you think?

Reply:
Yup 👍